packages/office-ui-fabric-react/src/components/DatePicker/DatePicker.test.tsx

```diff
@@ -93,7 +93,8 @@ describe('DatePicker', () => {
     datePicker.unmount();
   });
 
-  it('should call onSelectDate only once when allowTextInput is true and popup is used to select the value', () => {
+  // @todo: usage of document.querySelector is incorrectly testing DOM mounted by previous tests and needs to be fixed.
```

Comment:
:+1: Love this.

I know @ecraig12345 has been fixing this class of issues.. (and we also have the test-utils package that is supposed to help here that @dzearing checked in)...

Reply:
Yea, `safeMount` would alleviate this problem. This might be the motivation to refactor tests I needed. Its super easy to make this mistake.